### DISCUSSION TITLE
Gray default icons in Tool Library

### DISCUSSION THREAD

**Discussion by rappen** [MscrmTools/XrmToolBox#1367] [General]
I'm curious why some have this black/white default icon. Why is that?

![image](https://github.com/user-attachments/assets/b10d56c3-43e9-4eaf-9858-e68f222f2bed)

**Comment by MscrmTools** [comment]
Because they did not put any icon on their Nuget package

**Comment by rappen** [comment]
Right, now I know!
We should probably check that as well when validating new tools...
